### ISSUE TITLE
How to update failed test results

### ISSUE THREAD

**Issue by davyrouillard** [cisco/libacvp#513]
Hi, 
Sorry for this basic question but I does not found the answer in other issue or documentation.
When a bad test result is uploaded on the server (using --upload_vector), the status of test session is changed to "fail". When I try to upload a corrected test results, I obtain the following error:
```
    "error": "Validation error(s) on JSON payload.",
    "context": [
      "Vector set not in "processed" status.",
      "Unable to submit results when vector set in "failed" status."
```
 
Which command must be used to send update results ?
Thanks you
Davy

**Comment by abkarcher** [comment]
Hello,

Looking at the spec, it seems that resubmission needs to be done with a PUT instead of a POST. this is not something we have supported currently - but I hope to have it in the next release/soon.

Thanks,
Andrew

**Comment by abkarcher** [comment]
Addressed in #516. Branch libacvp_1_2_0-throttle now will automatically detect when vectors have previously been uploaded and re-upload them.  just run --vector_upload again as usual. Note that NIST determines whether or not this functionality is supported on the server end - I believe it is on the demo server but not on production. Please let us know if you experience any issues with this!

Thanks,
Andrew
